src/block/Anvil.php

```diff
@@ -100,10 +101,10 @@ public function tickFalling() : ?Block{
 		return null;
 	}
 
-	public function onHitGround(int $fallDistance, int $fallTime, AxisAlignedBB $boundsOnDespawn) : void{
-		foreach($this->position->getWorld()->getCollidingEntities($boundsOnDespawn) as $ent){
+	public function onHitGround(FallingBlock $blockEntity) : void{
+		foreach($this->position->getWorld()->getCollidingEntities($blockEntity->getBoundingBox()) as $ent){
 			if($ent instanceof Living){
-				$damageSource = new EntityDamageByBlockEvent($this, $ent, EntityDamageEvent::CAUSE_FALLING_BLOCK, min($fallDistance * 2, 40));
+				$damageSource = new EntityDamageByEntityEvent($blockEntity, $ent, EntityDamageEvent::CAUSE_FALLING_BLOCK, min($blockEntity->fallDistance * 2, 40));
```

Comment:
I have been thinking about the damage calculation of this. It appears the minimum fall distance (from survival tests) to do damage is >1 block (~1.3 damages you, tested by standing on an enchanting table). So instead of the min() there should also be a check for this.

Reply:
Added a check for making sure drop is >1 block